test/application/UtilsTest.php

```diff
@@ -552,4 +552,23 @@ public function GetMentionedObjectsFromTextProvider(): array
 			],
 		];
 	}
+
+	/**
+	 * @param string $sExpressionToConvert
+	 * @param int $iExpectedConvertedValue
+	 *
+	 * @dataProvider ConvertToBytesProvider
+	 */
+	public function testConvertToBytes($sExpressionToConvert, $iExpectedConvertedValue)
+	{
+		$iCurrentConvertedValue = utils::ConvertToBytes($sExpressionToConvert);
+		self::assertEquals($iExpectedConvertedValue, $iCurrentConvertedValue, 'Converted value wasn\'t the one expected !');
+	}
+
+	public function ConvertToBytesProvider()
+	{
+		return [
+			'512M' => ['512M', 512 * 1024 * 1024],
```

Comment:
Also add test for other options:
```suggestion
			'123' => ['123', 123],
			'56k' => ['56k', 56 * 1024],
			'512M' => ['512M', 512 * 1024 * 1024],
			'2G' => ['2G', 2 * 1024 * 1024 * 1024],
```